frontend/cypress/integration/talentsearch/submit-application-workflows.spec.js

```diff
@@ -0,0 +1,72 @@
+import { aliasMutation, aliasQuery } from "../../support/graphql-test-utils";
+
+describe("Submit Workflow Tests", () => {
+
+  beforeEach(() => {
+    // reseed database to ensure no existing application
+    cy.exec('php ../api/artisan migrate:fresh --seed');
+
+    // register queries
+    cy.intercept("POST", "/graphql", (req) => {
+      aliasQuery(req, "browsePools");
+      aliasQuery(req, "getPoolAdvertisement")
+      aliasQuery(req, "getReviewMyApplicationPageData");
+      aliasQuery(req, "getApplicationData");
+      aliasQuery(req, "MyApplications");
+
+      aliasMutation(req, "createApplication");
+      aliasMutation(req, "submitApplication");
+    });
+
+    cy.visit("/en/search");
+  });
+
+  it("Submits an application to the CMO pool", () => {
+    cy.login("applicant");
+    cy.visit("/en/browse/pools");
+
+    // Browse pools page
+    cy.wait("@gqlbrowsePoolsQuery");
```

Comment:
I get stuck here. It gets time out.
![Submit Workflow Tests -- Submits an application to the CMO pool (failed)](https://user-images.githubusercontent.com/89482910/191842675-6be48bd1-6e64-443d-9c04-c3d1c908ed6a.png)